Makefile.am

```diff
@@ -367,7 +367,8 @@ container-iso-build:
 	sudo $(CONTAINER_ENGINE) run \
 	--rm \
 	--privileged \
-	--tmpfs /var/tmp:rw,mode=1777 \
+	$(shell test $(shell grep MemTotal /proc/meminfo | awk '{print $$2}') -gt 7900000 && \
```

Comment:
Is the 7.9 number magic?

Reply:
This could be testable in the VM.

Reply:
pushed a change raising it to 15000000, just shy of 16GB

this change disallows `--tmpfs /var/tmp:rw,mode=1777` if you have less memory than 16GB.

the `--tmpfs /var/tmp:rw,mode=1777` option failed when tested on:
4GB x86 laptop - MemTotal 3800000 kB
8GB ec2 t2.large  - MemTotal: 7866840 kB

it worked on
16GB ec2 t2.xlarge - MemTotal: 16112088 kB

I can’t get an ec2 instance with more than 8GB and less than 16GB on which to test for a lower number.

15000000 gives some wiggle room for confirming 16GB ram on the build machine. It’s still a guess, I don’t know the true floor for that number.

Reply:
Wow it really needs 16GB for this?

I guess it depends on the media because you are storing the ISO into memory.

Reply:
So I tested `master` version on VM with 12GB memory and I was watching `watch -n 2 free -h` and the highest RAM consumption was 6.2GiB RAM. The issue we are fighting here is that Total Memory doesn't mean free memory and it depends what else you are doing on the system.

On the other hand this is only optimization which if missing will make the build only slower not broken. I would suggest to set the limit to 12GB RAM to be sure. We should be fine with less but this will avoid broken builds.